capstone-app/src/app/modules/photos/components/photos-section/photos-section.component.html

```diff
@@ -2,7 +2,7 @@
   <span *ngFor="let item of navItems">
     <span *ngIf="item.children && item.children.length > 0">
       <ng-template matMenuContent></ng-template>
-      <button mat-raised-button color="primary" [matMenuTriggerFor]="menu.childMenu || defaultMenu">
+      <button mat-raised-button color="grey" [matMenuTriggerFor]="menu.childMenu || defaultMenu">
```

Comment:
Does this work as intended? This is not one of the acceptable color inputs for mat-button.

Reply:
Yes, it does. It looks very nice in my opinion. Can I keep it if it works, or should I change it?